styles/globals.css

```diff
@@ -0,0 +1,332 @@
+  background-color: var(--primary-blue);
+}
+
+button.primary:hover {
+  background-color: #374399;
+}
+
+button.primary:active {
+  background-color: #002167;
+}
+
+button.primary.outline {
+  color: var(--primary-blue);
+  background-color: #eeeff4;
+  border-color: var(--primary-blue);
+}
+
+button.primary.outline:hover {
+  color: #374399;
+  border-color: #374399;
+}
+
+button.primary.outline:active {
+  color: #002167;
+  border-color: #002167;
+}
+
+button.secundary {
+  background-color: #0d712e;
+}
+
+button.secundary:hover {
+  background-color: #348145;
+}
+
+button.secundary:active {
+  background-color: #00450e;
+}
+
+button.secundary.outline {
+  color: #0d712e;
+  background-color: #eeeff4;
+  border-color: #0d712e;
+}
+
+button.secundary.outline:hover {
+  color: #348145;
+  border-color: #348145;
+}
+
+button.secundary.outline:active {
+  color: #00450e;
+  border-color: #00450e;
+}
+
+button:disabled {
+  color: #39393d;
+  background-color: var(--neutral-medium);
+  border: unset;
+}
+
+button.outline:disabled {
+  color: #989799;
+  background-color: #f5f5f5;
+  border: 1px solid #989799;
+}
+
+.no-margin {
+  margin: 0;
+}
+
+.margin {
+  margin: 16px;
+}
+
+.horizontal-margin {
+  margin: 0 16px;
+}
+
+.vertical-margin {
+  margin: 16px 0;
+}
+
+.no-padding {
+  padding: 0;
+}
+
+.padding {
+  padding: 16px;
+}
+
+.horizontal-padding {
+  padding: 0 16px;
+}
+
+.vertical-padding {
+  padding: 16px 0;
+}
+
+.flex-row-start {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+}
+
+.flex-row-end {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+}
+
+.flex-row-center {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.flex-row-space-between {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.flex-row-space-around {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.flex-row-space-evenly {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+/* https://www.browserstack.com/guide/what-are-css-and-media-query-breakpoints */
+
+/* Extra small devices (phones, 600px and down) */
+@media only screen and (max-width: 600px) {
+  * {
+    font-size: 28px;
+  }
+
+  .display {
+    font-size: 50px;
+  }
+
+  h1 {
+    font-size: 56px;
+  }
+
+  h2 {
+    font-size: 48px;
+  }
+
+  h3 {
+    font-size: 40px;
+  }
+
+  h4 {
+    font-size: 32px;
+  }
+
+  h5 {
+    font-size: 24px;
+  }
+
+  .b1 {
+    font-size: 28px;
+  }
+
+  .b2 {
+    font-size: 18px;
+  }
+
+  .b3 {
+    font-size: 16px;
+  }
+
+  .b4 {
+    font-size: 16px;
+  }
+
+  .b5 {
+    font-size: 14px;
+  }
+
+  button,
+  a {
+    font-size: 20px;
+  }
+}
+
+/* Small devices (portrait tablets and large phones, 600px and up) */
+/* @media only screen and (min-width: 600px) {
```

Comment:
If is not in use, you can remove the comments

Reply:
Done